pkg/servicecentral/client.go

```diff
@@ -389,3 +444,20 @@ func convertV2ServiceToV1(v2Service V2Service) ServiceData {
 	service.CreationTimestamp = &creationTimestamp
 	return service
 }
+
+func findOpsGenieTeamServiceAttribute(attributes []ServiceAttributeResponse) (s ServiceAttribute, found bool, err error) {
+	const opsGenieSchemaName = "opsgenie"
+	for _, attr := range attributes {
+		if attr.Schema.Name != opsGenieSchemaName {
+			continue
+		}
+
+		team, ok := attr.Value["team"]
+		if !ok {
+			return ServiceAttribute{}, false, errors.Errorf("expected to find team name within schema of name %q", opsGenieSchemaName)
+		}
+
+		return ServiceAttribute{Team: team}, true, nil
```

Comment:
It looks like this selects the first OpsGenie attribute, but I wonder if the DB doesn't still contain records that have multiple OpsGenie attributes? 🤷‍♂️